lld/ELF/Config.h

```diff
@@ -337,7 +337,7 @@ struct Configuration {
       return;
     cheriABI = true;
     if (dynamicLinker.empty())
-      dynamicLinker = "/libexec/ld-cheri-elf.so.1";
+      dynamicLinker = "/libexec/ld-elf.so.1";
```

Comment:
I'm not sure if this is needed at all anymore. I can test next week.

Reply:
Things seem to work with this removed entirely so I've done so.